### PR TITLE
[Merge for July 13 deploy] VACMS-5669: Update URL patterns for Leadership Listing nodes. 

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #5669

## Testing done
Visual, cypress, bulk update

## QA steps
- As an administrator, run bulk alias update on all content.
- Visit any location listing, and visually verify that the path is inherited from the parent item in the breadcrumb.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/node.type.page.yml [hunk 1]
```diff
@@ -27,6 +28,9 @@ third_party_settings:
     minimum_revisions_to_keep: 200
     minimum_age_to_delete: 0
     when_to_delete: 0
+  menu_force:
+    menu_force: 1
```
Comment (kevwalsh) [inline config/sync/node.type.page.yml:32]: I think we need to pull out this content type for now, [based on this slack convo](https://dsva.slack.com/archives/C01K37HRUAH/p1624570534096400). Sorry!

File: docroot/modules/custom/va_gov_db/va_gov_db.install [hunk 2]
```diff
@@ -525,3 +525,23 @@ function va_gov_db_update_8027(&$sandbox) {
     '%count' => $count,
   ]);
 }
+
+/**
+ * Enable pathauto for vamc leadership_listing node types.
```
Comment (kevwalsh) [inline docroot/modules/custom/va_gov_db/va_gov_db.install:530]: We may need to put this in its own PR, or think about where this fits into the #5500 runbook. Let's chat about it.

**Review by kevwalsh — CHANGES_REQUESTED** [review]
Requesting one change for now (removing menu_force from `page`), we probably need to discuss the other things.

File: config/sync/node.type.page.yml (hunk 1 above)
Comment (ethanteague) [reply config/sync/node.type.page.yml:32]: Np, a'tall.

Comment (kevwalsh) [reply config/sync/node.type.page.yml]: Crap, i just realized that i should have put this in #5689, the branch on which this branch is built. My bad. Let's fix it there, merge and rebase, and we can the rest of this merged next week.

File: config/sync/node.type.landing_page.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ langcode: en
 status: true
 dependencies:
   module:
+    - menu_force
```
Comment (kevwalsh) [inline config/sync/node.type.landing_page.yml:6]: Sorry let's remove this content type too, since we're also removing `page`, which is closely connected.

**Comment by kevwalsh** [comment]
i just put “do not merge” on it. because it resets the URL aliases to use “generate automatically”, this needs to be merged right before update all the Leadership Listing aliases. it’ll probably sit here for a week or so

File: docroot/modules/custom/va_gov_backend/va_gov_backend.module [hunk 4]
```diff
@@ -771,7 +771,9 @@ function _va_gov_backend_unset_fields(EntityInterface $entity) {
 function _va_gov_backend_get_unlocked_alias_bundles() {
 
   // Insert into array bundles that do not need alias locked upon publish.
-  return ['health_care_region_detail_page'];
+  return [
+    'leadership_listing',
```
Comment (kevwalsh) [inline docroot/modules/custom/va_gov_backend/va_gov_backend.module:775]: Not sure if we actually want this.Per slack

> we want to reset Leadership nodes to generate URLs from h1s, but then after we regenerate those h1s, we want it to go back to being locked again

**Comment by kevwalsh** [comment]
@ethanteague can you have a look at this confict?

**Comment by ethanteague** [comment]
> @ethanteague can you have a look at this confict?

Fixed, @kevwalsh

**Comment by kevwalsh** [comment]
Would be good to merge this Tuesday July 13 so that we can finish out #5500 that afternoon or the following morning.

**Review by kevwalsh — APPROVED** [review]
I thought i had approved this earlier.

**Pull request merged**